G++ and Clang build failing on Ubuntu

```bash
Using worker: worker-linux-docker-a995f850.prod.travis-ci.org:travis-linux-6
system_info
Build system information
Build language: cpp
Build group: stable
Build dist: precise
Build image provisioning date and time
Thu Feb  5 15:09:33 UTC 2015
Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 12.04.5 LTS
Release:	12.04
Codename:	precise
Linux Version
3.13.0-29-generic
Cookbooks Version
a68419e https://github.com/travis-ci/travis-cookbooks/tree/a68419e
GCC version
gcc (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3
Copyright (C) 2011 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
LLVM version
clang version 3.4 (tags/RELEASE_34/final)
Target: x86_64-unknown-linux-gnu
Thread model: posix
Pre-installed Ruby versions
ruby-1.9.3-p551
Pre-installed Node.js versions
v0.10.36
Pre-installed Go versions
1.4.1
Redis version
redis-server 2.8.19
riak version
2.0.2
MongoDB version
MongoDB 2.4.12
CouchDB version
couchdb 1.6.1
Neo4j version
1.9.4
RabbitMQ Version
3.4.3
ElasticSearch version
1.4.0
Installed Sphinx versions
2.0.10
2.1.9
2.2.6
Default Sphinx version
2.2.6
Installed Firefox version
firefox 31.0esr
PhantomJS version
1.9.8
ant -version
Apache Ant(TM) version 1.8.2 compiled on December 3 2011
mvn -version
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T17:29:23+00:00)
Maven home: /usr/local/maven
Java version: 1.7.0_76, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-7-oracle/jre
Default locale: en_US, platform encoding: ANSI_X3.4-1968
OS name: "linux", version: "3.13.0-29-generic", arch: "amd64", family: "unix"
git.checkout
0.80s$ git clone --depth=50 --branch=devel https://github.com/saifthe1/OpenFOAM-Windows.git saifthe1/OpenFOAM-Windows
Cloning into 'saifthe1/OpenFOAM-Windows'...
remote: Counting objects: 594, done.
remote: Compressing objects: 100% (329/329), done.
remote: Total 594 (delta 250), reused 556 (delta 212), pack-reused 0
Receiving objects: 100% (594/594), 1.55 MiB | 0 bytes/s, done.
Resolving deltas: 100% (250/250), done.
Checking connectivity... done.
$ cd saifthe1/OpenFOAM-Windows
$ git checkout -qf 0679362a7ed863e75ea63b0fa9fb6b4ae505a22e
This job is running on container-based infrastructure, which does not allow use of 'sudo', setuid and setguid executables.
If you require sudo, add 'sudo: required' to your .travis.yml
See http://docs.travis-ci.com/user/workers/container-based-infrastructure/ for details.
$ export CXX=g++
$ export CC=gcc
$ g++ --version
g++ (Ubuntu/Linaro 4.6.3-1ubuntu5) 4.6.3
Copyright (C) 2011 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
0.00s$ mkdir _build && cd _build
The command "mkdir _build && cd _build" exited with 0.
0.66s$ cmake ../
-- The C compiler identification is GNU
-- The CXX compiler identification is GNU
-- Check for working C compiler: /usr/bin/gcc
-- Check for working C compiler: /usr/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/g++
-- Check for working CXX compiler: /usr/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Looking for include files CMAKE_HAVE_PTHREAD_H
-- Looking for include files CMAKE_HAVE_PTHREAD_H - found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE 
-- Configuring done
CMake Error at libraries/CMake/GTest/CMakeLists.txt:22 (ADD_LIBRARY):
  Cannot find source file:
    /home/travis/build/saifthe1/OpenFOAM-Windows/libraries/CMake/GTest/../../Gtest/fused-src/gtest/gtest.h
  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx
-- Build files have been written to: /home/travis/build/saifthe1/OpenFOAM-Windows/_build
The command "cmake ../" exited with 1.
0.00s$ make
make: *** No targets specified and no makefile found.  Stop.
The command "make" exited with 2.
0.00s$ make test
make: *** No rule to make target `test'.  Stop.
The command "make test" exited with 2.
Done. Your build exited with 1.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,6 +9,7 @@ branches:
     only:
         - master
         - devel
+        - ubuntu-fix
 script:
     - mkdir _build && cd _build
     - cmake ../

--- a/libraries/CMake/GTest/CMakeLists.txt
+++ b/libraries/CMake/GTest/CMakeLists.txt
@@ -1,11 +1,10 @@
 ##################################
 #set GTEST_ROOT variable to call googletest folder in previous 
 # directory
-SET(GTEST_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/../../Gtest")
+SET(GTEST_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/../../GTest")
 # Use a variable to point to the root of googlemock within
 # the External directory.  This makes it easier to update
 # to new releases as desired.
-
 INCLUDE_DIRECTORIES( ${GTEST_ROOT}/fused-src )
 
 # Make the file lists.

